Terminal.Gui/ViewBase/View.Drawing.cs

```diff
@@ -151,6 +173,11 @@ private void DoDrawAdornmentsSubViews ()
             SetClip (saved);
         }
 
+        if (Padding?.NeedsLayout == true)
+        {
+            Padding.Layout ();
```

Comment:
Do not call layout from the draw loop. 

Please remove these calls to Layout. We should never do layout functions from within the draw path.

Reply:
Done in https://github.com/gui-cs/Terminal.Gui/pull/4210/commits/dd7c73fb2409cd1016be31827733270ce876fc26.